runelite-client/src/main/java/net/runelite/client/plugins/inventorytags/InventoryTagsOverlay.java

```diff
@@ -63,8 +64,29 @@ public void renderItemOverlay(Graphics2D graphics, int itemId, WidgetItem itemWi
 				Rectangle bounds = itemWidget.getCanvasBounds();
 				if (displayMode == DisplayMode.OUTLINE)
 				{
-					final BufferedImage outline = itemManager.getItemOutline(itemId, itemWidget.getQuantity(), color);
+					final int quantity = itemWidget.getQuantity();
+					final BufferedImage outline = itemManager.getItemOutline(itemId, quantity, color);
 					graphics.drawImage(outline, (int) bounds.getX(), (int) bounds.getY(), null);
+					if (itemManager.getItemComposition(itemId).isStackable())
+					{
+						graphics.setFont(FontManager.getRunescapeSmallFont());
+						String quantityText = Integer.toString(quantity);
```

Comment:
This strikes me as something which should be added to QuantityFormatter's API instead of only being used locally here.